test/testutils/syncenginetestutils.h

```diff
@@ -628,7 +628,7 @@ inline void addFiles(QStringList &dest, const FileInfo &fi)
         for (const auto &fi : fi.children)
             addFiles(dest, fi);
     } else {
-        dest += QStringLiteral("%1 - %2 %3-bytes").arg(fi.path()).arg(fi.contentSize).arg(fi.contentChar);
+        dest += QStringLiteral("%1 - %2 %3-bytes (%4)").arg(fi.path()).arg(fi.contentSize).arg(fi.contentChar).arg(fi.lastModifiedInUtc());
```

Comment:
.arg(, , , , )

Also the build seems to fail with this one.

Reply:
done